src/index.html

```diff
@@ -19,7 +20,7 @@
       <div class="container">
         <div class="header__content">
           <div class="header__top">
-            <a href="/" class="header__logo"></a>
+            <a href="#" class="header__logo"></a>
             <div class="header__top-action">
               <a href="tel: +1 212-923-3700" class="header__top-call icon"></a>
               <div class="header__top-call-hover call-hover">
```

Comment:
In 'header__top-call-hover'  - 'hover' is modifier?
It might be better to specify this explicitly with help '--'